src/biguint.rs

```diff
@@ -2120,6 +2120,21 @@ impl BigUint {
         }
     }
 
+    /// Returns the u32 digits representation of the `BigUint` in little-endian order.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use num_bigint::BigUint;
+    ///
+    /// let i = BigUint::parse_bytes(b"1125", 10).unwrap();
+    /// assert_eq!(i.to_u32_digits(), vec![1125]);
```

Comment:
Can you include examples with multiple digits to demonstrate the order? Even just `i << 32` should make this easy to see.